### PR TITLE
Add trigger customization docs

### PR DESCRIPTION
**Pull request by gab-satchi**

### CODE REVIEW

File: docs/eventing/broker/rabbitmq-broker/README.md [hunk 1]
```diff
@@ -126,6 +126,22 @@ For more information about configuring the `RabbitmqCluster` CRD, see the
...
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: high-throughput-trigger
+  annotations:
+    rabbitmq.eventing.knative.dev/parallelism: "10"
+spec:
+...
+```
```
Comment (abrennan89) [inline docs/eventing/broker/rabbitmq-broker/README.md:143]:
```suggestion
## Configure message ordering

By default, Triggers will consume messages one at a time to preserve ordering. If ordering of events isn't important and higher performance is desired, you can configure this by using the
`parallelism` annotation. Setting `parallelism` to `n` creates `n` workers for the Trigger that will all consume messages in parallel.

The following YAML shows an example of a Trigger with parallelism set to `10`:

```yaml
apiVersion: eventing.knative.dev/v1
kind: Trigger
metadata:
  name: high-throughput-trigger
  annotations:
    rabbitmq.eventing.knative.dev/parallelism: "10"
...
```
```

**Review by abrennan89 — COMMENTED** [review]
Some suggestions for readability and formatting

**Comment by gab-satchi** [comment]
Thanks for the quick review :)

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Pull request merged**